nexus/types/src/deployment.rs

```diff
@@ -1467,6 +1469,40 @@ mod table_display {
             builder.build()
         }
 
+        pub(super) fn make_physical_disk_table(&self) -> Table {
```

Comment:
In my somewhat loose idea of a new way to render blueprints, instead of using a builder to manually create a table, we'd instead return a data structure full of strings and metadata that would allow automatic rendering in a variety of output styles.